### PR TITLE
Add missing Expression Language values

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adding Auth2/OIDC custom claim-specific expressions.
- **Is this PR related to a Monolith release?** No.

### Resolves:

* [OKTA-387139](https://oktainc.atlassian.net/browse/OKTA-387139)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md
Comment (shantanusardal-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:374]: This note suggests you can use explicit references to app, like zendesk_9ao1g13.clientId.

However, earlier in the document there is a note which says `Note: Explicit references to apps aren't supported for custom OAuth/OIDC claims.`

Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md]: At your suggestion, I'll remove the note. Thanks!

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/create-claims/index.md
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/create-claims/index.md:17]:
```suggestion
      * See [Okta Expression Language](/docs/reference/okta-expression-language)
```

Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/customize-authz-server/create-claims/index.md:17]: From the InfoDev style guide, we just have to list the reference (unless there is a specific section in the topic that we are referring to): https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/722929640/InfoDev+Style+Guide#4.-Best-practices-for-InfoDev-styles

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/add-custom-claim/index.md
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/add-custom-claim/index.md:24]:
```suggestion
    > **Note:** Some expressions are custom claim-specific. See [Expressions for custom claims](/docs/reference/okta-expression-language/#expressions-for-custom-claims).
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute/index.md
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute/index.md:5]:
```suggestion
If you want to include certain app-specific information in a token claim, you can do so by first adding the metadata into the profile section of the app. You can access any values that are put inside the app profile using `app.profile` written in Okta Expression Language. See [Expressions for custom claims](/docs/reference/okta-expression-language/#expressions-for-custom-claims).
```

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:371]:
```suggestion
* See [Create claims](/docs/guides/customize-authz-server/create-claims)
```

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:372]:
```suggestion
* See [Include app-specific information in a custom claim](/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute)
```

**Review by vanngo-okta — COMMENTED** [review]
A few nits. Looks good!

**Review by vanngo-okta — APPROVED** [review]
LGTM! 🚢

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md
Comment (shantanusardal-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:381]: `Sample using app attributes` instead of `Sample using an app rule`. `rule` has special meaning in Okta, for example - group rules. But it does not apply here.

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md
Comment (shantanusardal-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:387]: **Sample using `access.scope`** instead of **Sample using an app rule**.

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md
Comment (shantanusardal-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:382]: To `include` and app Profile label, instead of `catch`

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md
Comment (shantanusardal-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:388]: To `include` a granted scope array, instead of `catch`

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md
Comment (shantanusardal-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:368]: Expressions for custom OAuth2/OIDC claims

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md
Comment (shantanusardal-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:370]: Oauth2/OIDC custom claims

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md
Comment (shantanusardal-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:388]: You can name a claim `scope` in API Access Management custom authorization servers. You can then use the EL expression `access.scope` in custom claims to return an array of granted scope strings.

To include a granted Scope array and convert it to a space-delimited string, use the following expression:<br>
`String.replace(Arrays.toCsvString(access.scope),","," ")`

Comment (shantanusardal-okta) [reply packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:388]: do we use scope with a capital S in other places in the documentation? I feel we should use scope instead of Scope unless you made it this way for a particular reason.

Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:388]: As we discussed, we might need to chat with Susan about capitalization of "scope", but I've remove the capital 'S' for now :)

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/create-claims/index.md
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/create-claims/index.md:15]:
```suggestion
    * **Value** &mdash; this option appears if you chose **Expression**. Use Okta Expression Language syntax to generate values derived from attributes in Universal Directory and app profiles, for example: `appuser.username`.
```

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:377]:
```suggestion
| `access.scope` | `access` refers to the access token that requests the scopes.<br>`scope` refers to the array of granted scopes. | access.scope |
```

**Review by susanharper-okta — APPROVED** [review]
just a couple of nits - approving so that I don't hold up this being merged after my comments are applied. thank you!

**Pull request merged**
